xcode/Subconscious/Shared/Components/BacklinksView.swift

```diff
@@ -10,7 +10,7 @@ import SwiftUI
 struct BacklinksView: View {
     var backlinks: [EntryStub]
     var onRequestDetail: (EntryLink) -> Void
-    var onLink: (_ context: ResolvedAddress, SubSlashlinkLink) -> Void
+    var onLink: (_ context: Peer, SubSlashlinkLink) -> Void
```

Comment:
I dropped the `ResolvedAddress` type in this PR. I think `Peer` is the right balance of flexible and legible.

Reply:
> Does passing Peer mean we will have to resolve peernames to DIDs more frequently in the code?

Technically yes but in practice using `Peer` allows us to provide a `DID` if we have it or fall back to a `Petname` when we do not. The issue is: when viewing a given note, any links in that note must be resolved to a DID at _some stage_ for us to visit it. For anything we've indexed (i.e. transcludes) we know the DID from the SQLite records but for any other link we either look it up now or later. The perf is the same.